recipes/alignoth/build.sh

```diff
@@ -1,11 +1,10 @@
 #!/bin/bash -e
 
-# TODO: Remove the following export when pinning is updated and we use
-#       {{ compiler('rust') }} in the recipe.
-export \
-    CARGO_NET_GIT_FETCH_WITH_CLI=true \
-    CARGO_HOME="${BUILD_PREFIX}/.cargo"
+export CFLAGS="${CFLAGS} -O3 -fno-rtti"
+export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include"
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
 
 export BINDGEN_EXTRA_CLANG_ARGS="${CFLAGS} ${CPPFLAGS} ${LDFLAGS}"
 
-cargo install --no-track --verbose --root "${PREFIX}" --path .
+RUST_BACKTRACE=1
+cargo install --no-track --locked --verbose --root "${PREFIX}" --path .
```

Comment:
Any comment, why the `--locked` is added here? Just trying to understand it.

Reply:
The `--locked` is preventing the installation to update any dependency. This was an issue here for example https://github.com/bioconda/bioconda-recipes/pull/53877.

Reply:
Ah, OK. So this basically forces it to use the project's `cargo.lock` file, right? That makes a lot of sense, leaving pinning up to the package's developer, a.k.a. as YOU in this case.